sdk/storage/azure-storage-blob/azure/storage/blob/_blob_client.py

```diff
@@ -131,6 +131,12 @@ class BlobClient(StorageAccountHostsMixin, StorageEncryptionMixin):  # pylint: d
         the exceeded part will be downloaded in chunks (could be parallel). Defaults to 32*1024*1024, or 32MB.
     :keyword int max_chunk_get_size: The maximum chunk size used for downloading a blob. Defaults to 4*1024*1024,
         or 4MB.
+    :keyword str version_id:
+            The version id parameter is an opaque DateTime
```

Comment:
nit: I think your indent if off here?
Also super nit: The line break after DateTime feels weird here. Can you bring it on to one line and then split where needed?